src/about.js

```diff
@@ -1,9 +1,69 @@
 import React from 'react';
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css"; 
+import "slick-carousel/slick/slick-theme.css";
 
-class About extends React.Component {
+
+class About  extends React.Component {
     render() {
+        const settings = {
+            dots: false,
+            infinite: true,
+            speed: 500,
+            slidesToShow: 4,
+            slidesToScroll: 1,
+            swipeToSlide: true,
+            arrows: true,
+            className: 'slides'
+        };
         return(
             <section id="about">
+                    <div className ="wire"></div>
+                    <Slider {...settings}>
+                        <div>
+                            <h3>1</h3>
+                        </div>
+                        <div>
+                            <h3>2</h3>
+                        </div>
+                        <div>
+                            <h3>3</h3>
+                        </div>
+                        <div>
+                            <h3>4</h3>
+                        </div>
+                        <div>
+                            <h3>5</h3>
+                        </div>
+                        <div>
+                            <h3>6</h3>
+                        </div>
+                    </Slider>
+                    <div id="header">
+                        <h2 id="title">What is a Hackathon?</h2>
+                    </div>
+                    <div id= "para">
```

Comment:
nit: space after equals shouldn't be there

Also, it'd be better if the id was a bit more descriptive than just `para`. What is the _content_ about?